src/dev-server/plane.ts

```diff
@@ -14,10 +15,19 @@ export type PlaneConnectResponse = {
   status_url: string
   status: string
 }
-export type PlaneStatusMessage = {
-  status: string,
-  time: number,
-}
+
+type PlaneTerminationReason = 'swept' | 'external' | 'keyexpired'
```

Comment:
Hmm, should I make this `key_expired` on the Plane side before we do this?

Reply:
I don't think we _have_ to wait for that. We aren't relying on "keyexpired" in this code, and this is also pinned to a version of Plane that formats it as "keyexpired". What do you think?

Reply:
Ok, I can lgtm this and then separately make the Plane change.